src/lib/user/actions/get-my-account.ts

```diff
@@ -0,0 +1,48 @@
+import { UserActor } from "../user-actor";
+import { dbQuery } from "@/lib/data/db-utils";
+import { MyAccount } from "../user-models";
+
+export async function getMyAccount(
+  actor: UserActor,
+): Promise<MyAccount | null> {
+  const { userId: user_id, dbPool, userMapper } = actor.getParams();
```

Comment:
Actually, if we don't retrieve userId in the SQL we don't need this awkward `userId: user_id` mapping.

Reply:
Done! b07f27143d693cce888d59d16c08072d117813f4